.circleci/config.yml

```diff
@@ -64,6 +64,14 @@ jobs:
           working_directory: wordpress-develop/src/wp-content/plugins/atomic-blocks/
           command: composer phpcs && npm run lint:js
 
+  e2e-tests:
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
This didn't work with `docker` instead of `machine`. That would be running Docker inside Docker 😄